api/turing/cluster/pvc.go

```diff
@@ -29,7 +29,7 @@ func (pvc *PersistentVolumeClaim) BuildPersistentVolumeClaim() *corev1.Persisten
 		},
 		Spec: corev1.PersistentVolumeClaimSpec{
 			AccessModes: accessModes,
-			Resources: corev1.ResourceRequirements{
+			Resources: corev1.VolumeResourceRequirements{
```

Comment:
The name of this struct has been renamed in newer versions of the k8s Go client.